libs/settings/data/src/lib/resource/settings.resource.ts

```diff
@@ -1,7 +1,22 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
+import { SettingsType } from '@architect-poc/settings/domain';
+import { Observable, of } from 'rxjs';
+import { MenuItem } from '../../../../domain/src/lib/menu-item.model';
+
+// helper function that simulates the response from the server
+const createMenuItemArray = (type: SettingsType): MenuItem[] => {
+  return new Array(10).fill(0).map((_, i) => ({
+    id: `${i}`,
+    name: `${type} menu item ${i}`,
+  }));
+};
```

Comment:
can you provide just a different set of items with proper names for each type?
e.g.:

**sidebar**:
* sidebar-settings
* layout
* ...

**spaces**:
* edit
* add user

in best case pick some real names actually visible available in cu